FashionConnect_ui/src/components/Collections/Collections.jsx

```
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import ProductCard from '../ProductCard/ProductCard';
+import './Collections.css';
+
+export default function Collections({ userId }) {
+    const [collections, setCollections] = useState([]);
+    const [products, setProducts] = useState([]);
+
+    useEffect(() => {
+        const fetchCollections = async () => {
+            try {
+                const user_collections = await fetch(`http://localhost:3000/collections/${userId}`);
+                const data = await user_collections.json();
+                setCollections(data);
+            } catch (error) {
+                console.error('Error fetching collections:', error);
```

You should probably rethrow the error here, otherwise you'll end up with an empty collections view?